templates/todo/task_list.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
```

Comment:
You can reuse this template to render tag and task listswith some flag that detect what you want to show